examples/contrib/cifar10/main.py

```diff
@@ -76,16 +76,21 @@ def training(local_rank, config):
     trainer = create_trainer(model, optimizer, criterion, lr_scheduler, train_loader.sampler, config, logger)
 
     # Let's now setup evaluator engine to perform model's validation and compute metrics
-    metrics = {
-        "Accuracy": Accuracy(),
-        "Loss": Loss(criterion),
-    }
+    metrics = {"Accuracy": Accuracy(), "Loss": Loss(criterion)}
 
     # We define two evaluators as they wont have exactly similar roles:
     # - `evaluator` will save the best model based on validation score
     evaluator = create_evaluator(model, metrics=metrics, config=config)
     train_evaluator = create_evaluator(model, metrics=metrics, config=config)
 
+    # Finding optimum Learning Rate using FastaiLRFinder
+    if config["lr_finder"] is True:
+        lr_finder = FastaiLRFinder()
+        to_save = {"model": model, "optimizer": optimizer}
+        with lr_finder.attach(trainer, to_save, diverge_th=5.0) as trainer_with_lr_finder:
+            trainer_with_lr_finder.run(train_loader)
+            lr_finder.apply_suggested_lr(optimizer)
```

Comment:
This should be outside the context manager.

Reply:
fixed!